src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -126,6 +126,7 @@ BtreeLBAManager::alloc_extent(
   extent_len_t len,
   paddr_t addr)
 {
+  ceph_assert((hint%(uint64_t)segment_manager.get_block_size()) == 0);
```

Comment:
Do we have an is_aligned() helper?

Reply:
we have #define assert_aligned(x) ceph_assert(((x)%ctx.tm.get_block_size()) == 0) defined in object_data_handler.cc, but it uses ctx, (lba doesn't have this info),  I also can define another macro,   but if only one place use it, seems not necessary.  So what do you think?

Reply:
Let's define a helper in seastore_types.h bool is_aligned(uint64_t offset, uint64_t alignment);